frontend/src/scenes/onboarding/sdks/feature-flags/angular.tsx

```diff
@@ -0,0 +1,16 @@
+import { SDKKey } from '~/types'
+
+import { NodeInstallSnippet, NodeSetupSnippet } from '../sdk-install-instructions'
+import { SDKInstallAngularInstructions } from '../sdk-install-instructions/angular'
+import { FlagImplementationSnippet } from './flagImplementationSnippet'
+import { LemonDivider } from 'lib/lemon-ui/LemonDivider'
+
+export function FeatureFlagsAngularInstructions(): JSX.Element {
+    return (
+        <>
+            <SDKInstallAngularInstructions />
+            <LemonDivider thick dashed className="my-4" />
+            <FlagImplementationSnippet sdkKey={SDKKey.JS_WEB} />
```

Comment:
I'm assuming the instructions are the same for angular and regular JS?

Reply:
yes!